src/compass/s1_cslc_qa.py

```diff
@@ -99,29 +112,47 @@ def raster_pixel_classification(self):
         '''
         self.classification_count_dict['topo'] = {}
         topo_dict = self.classification_count_dict['topo']
-        topo_dict['layover_percent'] = 0
-        topo_dict['shadow_percent'] = 0
-        topo_dict['combined_percent'] = 0
-        self.classification_count_dict['percent_land_pixels'] = 0
-        self.classification_count_dict['percent_valid_pixels'] = 0
+        topo_dict['percent_layover_pixels'] = value_description_dict(
+            0, 'Percentage of output pixels labeled layover')
+        topo_dict['percent_shadow_pixels'] = value_description_dict(
+            0, 'Percentage of output pixels labeled shadow')
+        topo_dict['percent_combined_pixels'] = value_description_dict(
+            0, 'Percentage of output pixels labeled layover and shadow')
+        self.classification_count_dict['percent_land_pixels'] = \
+            value_description_dict(
+                0, 'Percentage of output pixels labeld as land')
+        self.classification_count_dict['percent_valid_pixels'] = \
+            value_description_dict(
+                0, 'Percentage of output pixels are valid')
 
 
     def populate_rfi_dict(self):
         '''
         Place holder for populating SAFE RFI information
         '''
-        self.rfi_dict['is_rfi_info_available'] = True
+        self.rfi_dict['is_rfi_info_available'] = value_description_dict(
+            True, 'Whether or not RFI information is available')
         # Follow key/values only assigned if RFI info is avaiable
-        self.rfi_dict['rfi_mitigation_performed'] = True
-        self.rfi_dict['rfi_mitigation_domain'] = True
-        self.rfi_dict['rfi_burst_report'] = True
+        self.rfi_dict['rfi_mitigation_performed'] = value_description_dict(
+            True, 'Whether or not the RFI mitigation step was performed')
+        self.rfi_dict['rfi_mitigation_domain'] = value_description_dict(
+            True, 'Domain the RFI mitigation step was performed')
+        self.rfi_dict['rfi_burst_report'] = value_description_dict(
+            '', 'Burst RFI report')
+
 
 
-    def populate_orbit_dict(self):
+    def set_orbit_type(self, cfg):
         '''
-        Place holder for populating QA orbit information
+        Populate QA orbit information
         '''
-        pass
+        orbit_file_path = cfg.orbit_path[0]
+        if 'RESORB' in orbit_file_path:
```

Comment:
might wanna make it
```suggestion
        if 'RESORB' in Path(orbit_file_path).name:
```

using `from pathlib import Path` just in case there's some weirdness with a full path's directories